src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_COUNT = 5;
+    private static final int DEFAULT_VALUE = 10;
+    private static final Color COLOR_WHITE = Color.WHITE;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int figureNumber = random.nextInt(FIGURE_COUNT);
+        String color = colorSupplier.getRandomColor();
+
+        int radius = random.nextInt(FIGURE_COUNT);
```

Comment:
it's better to create separate private method for getting `random.nextInt(FIGURE_COUNT)` and use it for each type of figure (don't forget that private methods should be placed after all public mehtods)